sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -112,22 +112,38 @@ ios_events AS (
 ),
 ios_clients AS (
   SELECT
-    date(submission_timestamp) AS submission_date,
-    client_info.client_id,
-    `mozfun.norm.browser_version_info`(client_info.app_display_version) AS browser_version_info,
-    mozfun.glean.legacy_compatible_experiments(ping_info.experiments) AS experiments,
-    normalized_country_code AS country,
-    client_info.locale,
-    normalized_channel,
-    normalized_os_version,
-    sample_id
+    ARRAY_AGG(DATE(submission_timestamp) ORDER BY submission_timestamp DESC)[
+      OFFSET(0)
+    ] AS submission_date,
+    client_info.client_id AS client_id,
+    ARRAY_AGG(client_info.app_display_version ORDER BY submission_timestamp DESC)[
+      OFFSET(0)
+    ] AS browser_version_info,
+    mozfun.map.mode_last(
+      ARRAY_CONCAT_AGG(
+        mozfun.glean.legacy_compatible_experiments(ping_info.experiments)
+        ORDER BY
+          submission_timestamp
+      )
+    ) AS experiments,
```

Comment:
Oh except here, I guess, this seems quite a bit more complicated. What's happening here? I don't totally follow